omod/src/main/webapp/resources/pocforms/CareCardForm.html

```diff
@@ -147,63 +147,80 @@
             setValue('height.value', h);
             setValue('bp1.value', bp1);
             setValue('bp2.value', bp2);
-            setValue('bmi.value', bm.toFixed(2));
         });
 
 
 $(document).ready(function ()
 	{
-	   $('#height').keyup(calBMI);
-           $('#weight').keyup(calBMI);
-           $('#length').keyup(calBMI);
-	});
+    $('#height').keyup(calBMI);
+    $('#weight').keyup(calBMI);
+    $('#length').keyup(calBMI);
+});
 
 function calBMI(){
     var height_val = getValue('height.value');
     var weight_val = getValue('weight.value');
     var length_val = getValue('length.value');
+    if(height_val == 0){
```

Comment:
why is this being defaulted to 1

Reply:
Because division by zero shows "infinity" in the BMI textfield for by using keyup function